core/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+class Customer(models.Model):
+    first_name = models.CharField(max_length=200, null=True)
+    second_name = models.CharField(max_length=200, null=True)
+    phone_number = models.CharField(max_length=200, null=True)
+    email_address = models.CharField(max_length=200, null=True)
+    password = models.CharField(max_length=200, null=True)
+    date_created = models.DateTimeField(auto_now_add=True, null=True)
+
+    def __str__(self):
+        return self.first_name + (' ') + self.second_name
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=200, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Product(models.Model):
+    CATEGOTY = (('Indoor', 'Indoor'),
+                ('Out Door', 'Out Door'),
+    )
+    name = models.CharField(max_length=200, null=True)
+    price = models.FloatField(null=True)
+    category = models.CharField(max_length=200, null=True, choices=CATEGOTY)
+    description = models.CharField(max_length=200, null=True)
+    date_created = models.DateTimeField(auto_now_add=True, null=True)
+    tags =  models.ManyToManyField(Tag)
+
+    def __str__(self):
+        return self.name
+
+
+class Order(models.Model):
+    STATUS = (
+        ('Pending', 'Pending'),
```

Comment:
[The first element in each tuple is the actual value to be set on the model, and the second element is the human-readable name](https://docs.djangoproject.com/en/dev/ref/models/fields/#field-choices). 
So we can do something like this (or similar):
('pending', 'Pending'),
('out', 'Out for delivery'),
('delivered', 'Delivered'),